queries.sql

```diff
@@ -1,3 +1,11 @@
-/*Queries that provide answers to the questions from all projects.*/
+/* Queary database */
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR 
+name = 'Pikachu';
+SELECT name, escape_attempts FROM animals WHERE weight_kg <= 10.50;
```

Comment:
- [ ] In this query we have to list names and escape attempts of animals that weigh more than 10.5kg not smaller than and equal to 10.5kg. Please make sure you fix this query
  - You can consider the below code `SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.50;`

![image](https://user-images.githubusercontent.com/83803414/212717399-cb1b41af-1506-4c92-8a66-ebe4c7e210e5.png)